wforce/wforce-replication.hh

```diff
@@ -46,18 +46,20 @@ public:
   void setNumSiblingThreads(unsigned int num_threads) { d_num_sibling_threads = num_threads; }
   GlobalStateHolder<vector<shared_ptr<Sibling>>>& getSiblings() { return d_siblings; }
   void replicateOperation(const ReplicationOperation& rep_op);
+  void setEncryptionKey(const std::string& key) { d_key = key; }
+  std::string getEncryptionKey() { return d_key; }
 protected:
-  bool checkConnFromSibling(const ComboAddress& remote, shared_ptr<Sibling>& recv_sibling);
+  virtual bool checkConnFromSibling(const ComboAddress& remote, shared_ptr<Sibling>& recv_sibling);
   void parseTCPReplication(std::shared_ptr<Socket> sockp, const ComboAddress& remote, std::shared_ptr<Sibling> recv_sibling);
   void parseReceivedReplicationMsg(const std::string& msg, const ComboAddress& remote, std::shared_ptr<Sibling> recv_sibling);
-private:
```

Comment:
Was the removal of `private:` intended?

Reply:
yes it was - these all need to be accessed from a subclass.